### DISCUSSION TITLE
ListCompareAlgorithm.AS_SET only for some objects

### DISCUSSION THREAD

**Discussion by ArkadiuszBlejwas** [javers/javers#1074] [Q&A]
Is it possible to set ListCompareAlgorithm.AS_SET only for some objects that are part of some larger X object, not for the whole of that X object?

**Comment by bartoszwalacik** [comment]
please ask this kind of basic questions at https://stackoverflow.com/questions/tagged/javers
it doesn't looks like a topic for disqussion

**Reply by ArkadiuszBlejwas** [reply]
ok, https://stackoverflow.com/questions/66604719/javers-listcomparealgorithm-as-set-only-for-some-objects
